package.json

```diff
@@ -5,9 +5,11 @@
   "author": "webcompat.com contributors",
   "scripts": {
     "eslint": "eslint --ignore-path .eslintignore -c .eslintrc . --cache",
-    "run:firefox": "web-ext run --source-dir=dist/firefox",
+    "firefox": "web-ext run --source-dir=dist/firefox",
+    "fennec": "web-ext run --target=firefox-android --source-dir=dist/firefox-mobile --firefox-apk=org.mozilla.fennec_aurora",
```

Comment:
We should probably have `fennec-nightly` and `fennec`, running with `--firefox-apk=org.mozilla.firefox` so we can test this on stable Firefox for Android as well.

Reply:
yeah, good point. will do.